python/seldon_core/user_model.py

```diff
@@ -123,6 +123,18 @@ def init_metadata(self) -> Dict:
         raise SeldonNotImplementedError("init_metadata is not implemented")
 
 
+class SeldonPrediction:
```

Comment:
Name should change to something more specific, as this is returned in aggregate, transform_, etc

Reply:
How would you suggest to name it?

Reply:
Maybe `SeldonResponse` or `ClientResponse`.

Reply:
Went with `ClientResponse`, can still adjust easily